crates/cli/src/custom_section.rs

```diff
@@ -0,0 +1,16 @@
+use anyhow::Result;
+
+pub const SOURCE_CODE_SECTION_NAME: &str = "javy_source";
+
+pub fn compress_source_code(source_code: &[u8]) -> Result<Vec<u8>> {
+    let mut compressed_source_code: Vec<u8> = vec![];
+    brotli::enc::BrotliCompress(
+        &mut std::io::Cursor::new(source_code),
+        &mut compressed_source_code,
+        &brotli::enc::BrotliEncoderParams {
+            quality: 11,
+            ..Default::default()
+        },
+    )?;
+    Ok(compressed_source_code)
+}
```

Comment:
Can we change the name of this module to something else? Personally I feel that `custom_section` is not the most appropriate name, since it doesn't do anything custom section specific. Perhaps `compression` or something around those lines?

The only thing that is vaguely related to a custom section is `SOURCE_CODE_SECTION_NAME`, but that constant can live in `main.rs` which the module orchestrating the module creation.

Reply:
I opted to go with `source_code_section`